disnake/ext/commands/base_core.py

```diff
@@ -53,11 +76,6 @@
 HookT = TypeVar("HookT", bound="Hook")
 ErrorT = TypeVar("ErrorT", bound="Error")
 
-if TYPE_CHECKING:
-    P = ParamSpec("P")
-else:
-    P = TypeVar("P")
```

Comment:
This is no longer defined at runtime, not sure if that will be an issue.

Reply:
That was added in 6cb0f36dee410c47e4730d7adf73694382df512f, appears like it was never used (and definitely isn't being used at the moment).